protos/receipt.proto

```diff
@@ -0,0 +1,47 @@
+// Copyright 2018 Bitwise IO
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+syntax = "proto3";
+
+message Receipt {
+    enum PaymentType {
+        COUPON = 0;
+        REDEMPTION = 1;
+    }
+
+    // ID of the bond for which this payment occurred
+    string bond_id = 1;
+
+    // Ticker symbol of the organization to which the payment was paid
+    string payee = 2;
```

Comment:
Maybe this could be named `organization_id` or `payee_organization_id`? To keep it consistent across protos, and make it clear they are all referring to the same field in organization.